src/main/java/at/hannibal2/skyhanni/test/command/TrackSoundsCommand.kt

```diff
@@ -0,0 +1,50 @@
+package at.hannibal2.skyhanni.test.command
+
+
+import at.hannibal2.skyhanni.events.PlaySoundEvent
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.OSUtils
+import at.hannibal2.skyhanni.utils.SimpleTimeMark
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+import java.util.concurrent.atomic.AtomicBoolean
+import kotlin.time.Duration.Companion.seconds
+
+object TrackSoundsCommand {
+
+    var cutOfTime: SimpleTimeMark = SimpleTimeMark.farPast()
+
+    val sounds = mutableListOf<PlaySoundEvent>()
+
+    var enable = AtomicBoolean(true)
```

Comment:
There is no point using an `AtomicBoolean` here. An `AtomicBoolean` used only with `get` and `set` provides no benefit over the primitive. Either use some form of locking, or a CAS mechanism. Both of those are overkill (the worst case scenario is that you need to redo your command). Actually all of that is not needed since `DelayedRun` runs on the main thread.